app/pdfs/ticket_pdf.rb

```diff
@@ -51,7 +51,9 @@ def draw_second_square
     move_down 70
     draw_text @conference.title.to_s, at: [@mid_horizontal + 30, cursor - 30], size: 12
     draw_text @conference.organization.name.to_s, at: [@mid_horizontal + 30, cursor - 50], size: 12
-    draw_text @conference.venue.name.to_s, at: [@mid_horizontal + 30, cursor - 70]
+    if !@conference.venue.nil?
+      draw_text @conference.venue.name.to_s, at: [@mid_horizontal + 30, cursor - 70]
```

Comment:
`to_s` shouldn't be needed, right? `name` is already an string (I know this was already there, so you can add the change in a different commit) :wink:

Reply:
@Ana06 Sure! :smile: